testing/test_repo.py

```diff
@@ -11,7 +11,7 @@ def repo(home, bin):
 
 @pytest.mark.parametrize('package, glob', [
     ('grin', 'grin*'),
-    ('pipsi', 'pipsi*'),
+#    ('pipsi', 'pipsi*'),
```

Comment:
in order to keep track, please make this an `xfail` with a reason reference

Reply:
OK, will do. It's a bit fiddly to just mark one set of parameters as xfail. I think I have the correct incantation, but if it fails again I'll work out how to fix it.